src/azul/__init__.py

```diff
@@ -430,13 +424,18 @@ def drs_endpoint(self):
     def lambda_names(self) -> List[str]:
         return ['indexer', 'service']
 
-    @property
-    def indexer_name(self) -> str:
-        return self.qualified_resource_name('indexer')
+    def indexer_name(self, lambda_name: Optional[str] = None) -> str:
```

Comment:
```suggestion
    def indexer_name(self, handler_name: Optional[str] = None) -> str:
```

`handler_name` is the name of the handler function
`lambda_name` is `'indexer'` or `'service'`
`function_name` is the fully qualified name of the AWS Lambda function

and let's leave this as a property but add `indexer_function_name` and `service_function_name` methods that call …

Reply:
I like this renaming scheme, and calling the parameter `handler_name` makes sense. I'd gone with `lambda_name`, in part because that's the name of the property of the function, for example, `contribute.lambda_name`. Should I change the name of the property as well?

My plan for now is to not change the property name, and push to review as is.